_assets/css/_landing.scss

```diff
@@ -32,345 +32,352 @@ Stylesheet: Landing Page Stylesheet
-        );
-        align-items: flex-end;
-
-        &::after {
-          @include icon_font($char: $icon-arrow-right);
-        }
-      }
-
-      .bak {
-        left: 0;
-        background-image: linear-gradient(
-          to right,
-          rgba($white, 1) 50%,
-          rgba($white, 0) 100%
-        );
-        align-items: flex-start;
-
-        &::after {
-          @include icon_font($char: $icon-arrow-left);
-        }
-      }
-
-      &.scrollable {
-        .fwd,
-        .bak {
-          visibility: visible;
-          pointer-events: auto;
-        }
-
-        &.start .bak {
-          visibility: hidden;
-          pointer-events: none;
-        }
-
-        &.end .fwd {
-          visibility: hidden;
-          pointer-events: none;
-        }
-      }
-    }
-
-    &.sticky nav {
-      .group-a {
-        width: calc(100% - #{rem-calc(155)});
-
-        a {
-          font-size: rem-calc(13);
-
-          &.active::after {
-            top: rem-calc(38);
-          }
-        }
-      }
-
-      .group-b {
-        opacity: 1;
-        margin-right: 0;
-      }
-
-      .fwd {
-        right: rem-calc(150);
-      }
-    }
-  }
-
-  @include breakpoint(small down) {
-    nav {
-      height: auto;
-      min-height: rem-calc(52);
-
-      .jump-link {
-        position: absolute;
-        z-index: 2;
-        left: 0;
-        top: 0;
-        height: rem-calc(52);
-        display: flex;
-        flex-flow: row nowrap;
-        justify-content: flex-start;
-        align-items: center;
-        padding: rem-calc(10 10 10 35);
-        width: calc(100% - #{rem-calc(155)});
-        font-weight: $global-weight-normal;
-        background-color: $white;
-        margin: 0;
-        cursor: pointer;
-
-        &::after {
-          @include icon_font($char: $icon-arrow-down);
-
-          display: block;
-          flex: 0 1 #{rem-calc(22)};
-          width: rem-calc(22);
-          padding: rem-calc(3 0 3 6);
-          font-size: rem-calc(16);
-        }
-      }
-
-      .group-a {
-        display: none;
-        z-index: 1;
-        height: auto;
-        position: relative;
-        margin-top: rem-calc(54);
-        padding: rem-calc(12 10);
-        border-top: 1px solid $medium-gray;
-
-        a {
-          display: block;
-          padding: rem-calc(8 0 8 25);
-          font-size: rem-calc(15);
-
-          &.active {
-            color: $body-font-color;
-
-            &::after {
-              content: '';
-              display: block;
-              position: absolute;
-              left: rem-calc(12);
-              top: rem-calc(11);
-              bottom: auto;
-              width: rem-calc(1);
-              height: rem-calc(16);
-              background-color: $primary-color;
-            }
-          }
-        }
-      }
-
-      .group-b {
-        z-index: 2;
-        opacity: 1;
-        margin-right: 0;
-        height: rem-calc(52);
-      }
+// .page-nav-container {
```

Comment:
If you don't think you're going to need this it's better to remove it out-right rather than comment it out. The old style will still be there in history if you need it.